apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -0,0 +1,260 @@
+const {
+    TOO_MUCH_READINGS,
+    ORGANIZATION_NOT_FOUND,
+    PROPERTY_NOT_FOUND,
+    INVALID_METER_VALUE,
+} = require('@condo/domains/meter/constants/errors')
+const { Meter, MeterReading } = require('@condo/domains/meter/utils/serverSchema')
+
+const READINGS_LIMIT = 500
+
+const ERRORS = {
+    TOO_MUCH_READINGS: (sentCount) => ({
+        code: BAD_USER_INPUT,
+        type: TOO_MUCH_READINGS,
+        message: `Too much readings. Maximum is ${READINGS_LIMIT}.`,
+        messageForUser: 'api.meter.registerMetersReadings.TOO_MUCH_READINGS',
+        messageInterpolation: { limit: READINGS_LIMIT, sentCount },
+    }),
+    ORGANIZATION_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: ORGANIZATION_NOT_FOUND,
+        message: 'Organization not found',
+        messageForUser: 'api.meter.registerMetersReadings.ORGANIZATION_NOT_FOUND',
+    },
+    PROPERTY_NOT_FOUND: {
+        code: BAD_USER_INPUT,
+        type: PROPERTY_NOT_FOUND,
+        message: 'Property not found',
+        messageForUser: 'api.meter.registerMetersReadings.PROPERTY_NOT_FOUND',
+    },
+    INVALID_METER_VALUE: (key, value) => ({
+        code: BAD_USER_INPUT,
+        type: INVALID_METER_VALUE,
+        message: 'Invalid meter value',
+        messageForUser: 'api.meter.registerMetersReadings.INVALID_METER_VALUE',
+        messageInterpolation: { key, value },
+    }),
+}
+
+function toISO (str) {
+    return dayjs(str).toISOString()
+}
+
+/**
+ * @param value
+ * @return {undefined|string|null}
+ */
+function normalizeMeterValue (value) {
+    if (!(isString(value) || isNumber(value) || isUndefined(value))) return null
+    if (isUndefined(value)) return value
+
+    const transformedValue = String(value).trim().replaceAll(',', '.')
+    if (isEmpty(transformedValue)) return undefined
+    return String(Number(transformedValue))
+}
+
+/**
+ * @param {string | null | undefined} value
+ * @return {boolean}
+ */
+function validateMeterValue (value) {
+    if (!isString(value) && !isUndefined(value)) return false
+    if (isUndefined(value)) return true
+
+    return !isEmpty(value) && !isNaN(Number(value)) && isFinite(Number(value)) && Number(value) >= 0
+}
+
+const RegisterMetersReadingsService = new GQLCustomSchema('RegisterMetersReadingsService', {
+    types: [
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsMeterMetaInput {' +
+                'numberOfTariffs: Int,' +
+                'place: String,' +
+                'verificationDate: String,' +
+                'nextVerificationDate: String,' +
+                'installationDate: String,' +
+                'commissioningDate: String,' +
+                'sealingDate: String,' +
+                'controlReadingsDate: String' +
+                '}',
+        },
+        {
+            access: true,
+            type: 'input RegisterMetersReadingsReadingAddressMetaInput { unitType: String!, unitName: String!, globalId: String }',
```

Comment:
Are you sure about naming. I guess its not good idea, when multiple thing have the same name

Reply:
What do you mean? Which thing has the same name?